sessiontxn/isolation/base.go

```diff
@@ -57,6 +57,7 @@ type baseTxnContextProvider struct {
 	infoSchema      infoschema.InfoSchema
 	txn             kv.Transaction
 	isTxnPrepared   bool
+	constStartTS    uint64
```

Comment:
Better to add some comments about the use case restrictions.

Reply:
Done